Traceback (most recent call last):
  File "/home/ubuntu/onsite-test-server/tester/logger.py", line 79, in wrapper
    result = func(func_cls, *args, **kwargs)
TypeError: func() takes 2 positional arguments but 3 were given
